### ISSUE TITLE
Drop xlink:href (and therefore xmlns:xlink) from all the SVG files

### ISSUE THREAD

**Issue by gijsk** [mozilla/activity-stream#2767]
Also see https://bugzilla.mozilla.org/show_bug.cgi?id=1359030 turned up:

```
data/content/assets/glyph-bookmark-16.svg
data/content/assets/glyph-bookmark-remove-16.svg
data/content/assets/glyph-delete-16.svg
data/content/assets/glyph-dismiss-16.svg
data/content/assets/glyph-more-16.svg
data/content/assets/glyph-newWindow-16.svg
data/content/assets/glyph-newWindow-private-16.svg
```

I think the xlink namespace declaration can be dropped from these, and we can use plain `href` attributes instead of `xlink:href` for referencing.

**Comment by Mardak** [comment]
This is not specific to the test pilot version. e.g., https://raw.githubusercontent.com/mozilla/activity-stream/master/system-addon/data/content/assets/glyph-bookmark-16.svg

**Comment by staabm** [comment]
it seems this is no longer a open todo. wasn't able to find any reference to `xlink` in the master branch.

**Comment by Mardak** [comment]
Some of this was fixed as part of #3198 (https://github.com/mozilla/activity-stream/pull/3198/commits/efed0365c6c7d4c30e576850a3ee2263baf513ff) and rest cleaned up in #3235 (https://github.com/mozilla/activity-stream/pull/3235/commits/a14ce2f760fc960d69108f7cfc0384d7e763355d)
